.rubocop.yml

```diff
@@ -1,11 +1,7 @@
 AllCops:
-  Include:
-    - 'Gemfile'
```

Comment:
Yeah it is see the commit message. Also: https://github.com/bbatsov/rubocop/blob/master/config/default.yml#L9-L27